email_split/__init__.py

```diff
@@ -0,0 +1,25 @@
+# Load in our dependencies
+import re
+
+
+# Define an email class
+class Email(object):
+    def __init__(self, local, domain):
+        # local is content before `@` in email
+        # domain is content after `@` in email
+        self.local = local
+        self.domain = domain
+
+    @classmethod
+    def split_email(cls, email):
+        """Break up an email and initialize a new class"""
+        # Break up our email at its `@`
+        # https://docs.python.org/2/library/re.html#re.split
+        email_parts = re.split(r'@', email, maxsplit=1)
```

Comment:
TIL about `partition`. Will update to use that https://docs.python.org/2/library/stdtypes.html#str.partition